dev-qt/qtwebengine/qtwebengine-5.11.1.recipe

```diff
@@ -0,0 +1,207 @@
+SUMMARY="Qt WebEngine rendering library for dynamic web content"
+DESCRIPTION="Qt WebEngine provides functionality for rendering regions of dynamic web content. \
+The functionality in Qt WebEngine is divided into the following modules: \
+* QtWebEngineCore: Provides public API shared by both QtWebEngine and QtWebEngineWidgets \
+* QtWebEngine: Provides QML types for rendering web content within a QML application \
+* QtWebEngineWidgets: Provides a web browser engine as well as C++ classes to render \
+and interact with web content"
+
+HOMEPAGE="https://www.qt.io"
+COPYRIGHT="2015-2018 The Qt Company Ltd."
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://download.qt.io/official_releases/qt/5.11/5.11.1/submodules/qtwebengine-everywhere-src-5.11.1.tar.xz"
+CHECKSUM_SHA256="389d9f42ca393ac11ec8932ce9771766dec91a4c761ffb685cc429c2a760d48c"
+SOURCE_DIR="qtwebengine-everywhere-src-5.11.1"
+PATCHES="qtwebengine-5.11.1.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	qtwebengine$secondaryArchSuffix = $portVersion compat >= 5
+	lib:libQt5WebEngine$secondaryArchSuffix = $portVersion compat >= 5
+	lib:libQt5WebEngineCore$secondaryArchSuffix = $portVersion compat >= 5
+	lib:libQt5WebEngineWidgets$secondaryArchSuffix = $portVersion compat >= 5
+	cmd:qwebengine_convert_dict$secondaryArchSuffix = $portVersion compat >= 5
```

Comment:
I strongly advise using `libVersion` convention shared by many recipes. More details can be found [here][0]

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#using-the-libversioncompat-user-variable